### PR TITLE
feat: add blog projects

### PR DESCRIPTION
**Pull request by rogerogers**
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.

image

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/en/blog/news/1st_RPCKitex/index.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
-keywords: ["CloudWeGo", "RPC framework", "Kitex", "microservice framework", "ByteDance Open Source", "open source"]
+keywords:
+  [
+    "CloudWeGo",
+    "RPC framework",
+    "Kitex",
+    "microservice framework",
+    "ByteDance Open Source",
+    "open source",
+  ]
```
Comment (GuangmingLuo) [inline content/en/blog/news/1st_RPCKitex/index.md:14]: 这个保持不变吧，和其他格式一致吧

File: content/en/blog/news/Kitex_Proxyless_OpenTelemetry/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 date: 2022-12-20
 title: "Kitex Proxyless Practice：Traffic Lane Implementation with Istio and OpenTelemetry"
+projects: ["Kitex", "Hertz"]
```
Comment (GuangmingLuo) [inline content/en/blog/news/Kitex_Proxyless_OpenTelemetry/index.md:4]: Kitex Only

File: content/en/blog/news/Introducing CloudWeGo/index.md [hunk 3]
```diff
@@ -1,75 +1,105 @@
...
+keywords:
+  [
+    "CloudWeGo",
+    "middleware",
+    "Kitex",
+    "microservice framework",
+    "ByteDance Open Source",
+    "open source",
+    "cloud native",
+  ]
```
Comment (GuangmingLuo) [inline content/en/blog/news/Introducing CloudWeGo/index.md:15]: remain the same

File: content/en/blog/news/bytedance_gonet_practice/index.md [hunk 4]
```diff
@@ -1,8 +1,10 @@
 ---
 date: 2020-05-24
 title: "ByteDance Practices on Go Network Library"
+projects: ["Netpoll"]
 linkTitle: "ByteDance Practices on Go Network Library"
-keywords: ["Netpoll", "Go", "epoll", "Network Library", "Multiplexing", "ZeroCopy"]
+keywords:
+  ["Netpoll", "Go", "epoll", "Network Library", "Multiplexing", "ZeroCopy"]
```
Comment (GuangmingLuo) [inline content/en/blog/news/bytedance_gonet_practice/index.md:7]: remain the same

File: content/en/blog/releases/Netpoll/_index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Netpoll Release"
 linkTitle: "Netpoll"
+projects: []
```
Comment (GuangmingLuo) [inline content/en/blog/releases/Netpoll/_index.md:4]: add Netpoll

File: content/zh/blog/news/Evolution_Kitex_HighPerformance/index.md [hunk 6]
```diff
@@ -1,8 +1,9 @@
 ---
 date: 2022-05-19
 title: "字节微服务框架的挑战和演进"
+projects: ["CloudWeGo", "Kitex"]
```
Comment (GuangmingLuo) [inline content/zh/blog/news/Evolution_Kitex_HighPerformance/index.md:4]: Kitex only

File: content/zh/blog/news/Kitex_Proxyless_OpenTelemetry/index.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 date: 2022-11-08
 title: "Kitex Proxyless 之流量路由：配合 Istio 与 OpenTelemetry 实现全链路泳道"
+projects: ["Kitex", "Hertz"]
```
Comment (GuangmingLuo) [inline content/zh/blog/news/Kitex_Proxyless_OpenTelemetry/index.md:4]: Kitex only

File: content/en/blog/news/Introducing CloudWeGo/index.md [hunk 8]
```diff
@@ -1,75 +1,105 @@
...
+  Kitex is equipped with built-in governance strategies and expansion interfaces for frictionless integration into the microservice system. Hertz is a Go HTTP framework with high-performance and strong-extensibility for building micro-services. Netpoll is aimed at scenarios that demand high performance on RPC scenarios. Shmipc is a high performance inter-process communication library, built on Linux's shared memory technology and uses unix or tcp connection to do process synchronization and finally implements zero copy communication across inter-processes. Volo is a high-performance and strong-extensibility Rust RPC framework that helps developers build microservices. Each component of CloudWeGo can be used separately.
```
Comment (GuangmingLuo) [inline content/en/blog/news/Introducing CloudWeGo/index.md:85]: extra space at the beginning ?

Comment (rogerogers) [reply content/en/blog/news/Introducing CloudWeGo/index.md]: 识别到列表里面了，我隔一行。

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**